client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -217,17 +243,25 @@ export default defineComponent({
     <v-dialog
       :value="pendingImportPayload !== null || importMultiCamDialog || checkingMedia"
       persistent
-      width="800"
+      :width="bulkImport ? 'auto' : '800'"
       overlay-opacity="0.95"
       max-width="80%"
     >
-      <ImportDialog
-        v-if="pendingImportPayload !== null"
-        :import-data="pendingImportPayload"
-        :disabled="importing"
-        @finalize-import="finalizeImport($event)"
-        @abort="pendingImportPayload = null"
-      />
+      <template v-if="pendingImportPayload !== null">
+        <BulkImportDialog
+          v-if="bulkImport"
+          :import-data="pendingImportPayload"
+          @finalize-import="finalizeBulkImport($event)"
+          @abort="pendingImportPayload = null"
+        />
+        <ImportDialog
+          v-else
+          :import-data="pendingImportPayload[0]"
```

Comment:
If your bulk import doesn't find any datasets it returns an array of length 0, not null or  something else so this won't work.